app/land-action/index.js

```diff
@@ -15,9 +15,46 @@ const getActions = () => {
   return actions
 }
 
+function addRule (action, newRule) {
+  if (!action.eligibilityRules) {
+    action.eligibilityRules = []
+  }
+  if (!action.eligibilityRules.some(rule => rule.id === newRule.id)) {
+    action.eligibilityRules.push(newRule)
+  }
+  const actionIndex = actions.findIndex(a => a.code === action.code)
```

Comment:
lines 25-28 are duplicated in the `updateRule` and `deleteRule` functions
Refactor and create a function to avoid the duplication

Reply:
done

Reply:
lines 25-28 can be deleted